desktop/cmp/form/inputs/NumberInput.js

```diff
@@ -157,6 +160,8 @@ export class NumberInput extends HoistInput {
     }
 
     parseValue(value) {
+        if (isNumber(value)) return value;
```

Comment:
If value is committed via the enter key the external and internal vals will both be a number. This method gets called again onBlur via hoist input's logic. We don't need to parse and the next line would throw.